doc/classes/Vector2.xml

```diff
@@ -299,7 +299,7 @@
 			<argument index="1" name="weight" type="float" />
 			<description>
 				Returns the result of spherical linear interpolation between this vector and [code]to[/code], by amount [code]weight[/code]. [code]weight[/code] is on the range of 0.0 to 1.0, representing the amount of interpolation.
-				[b]Note:[/b] Both vectors must be normalized.
+				This method also handles interpolating the lengths if the input vectors have different lengths.
 			</description>
```

Comment:
May possibly be worth mentioning the special behaviour if one or both is zero length?